src/index.d.ts

```diff
@@ -887,6 +887,13 @@ export class Input extends React.Component<InputProps, any> {
    * eg `this.inputRef.clear()`
    */
   clear(): void;
+
+  /**
+   * Calls setNativeProps on the Input
+   * 
+   * eg `this.inputRef.setNativeProps({ text: 'any text' })`
+   */
+  setNativeProps(nativeProps: Object): void;
```

Comment:
We could probably type this better.

```suggestion
  setNativeProps(nativeProps: Partial<TextInputProperties>): void;
```

Reply:
`setNativeProps (nativeProps: Partial <TextInputProps>): void;` not a better option? Because [TextInputProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bcedcaf254ac774013661e14b64d811f503076dd/types/react-native/legacy-properties.d.ts#L98) are deprecated.

Reply:
Sure go ahead, they’re technically the same thing. One aliases the other